### ISSUE TITLE
command/reboot.rb is missing from 0.5.0, breaking `vagrant help`.

### ISSUE THREAD

**Issue by chrisbaldauf** [tknerr/vagrant-managed-servers#32]
I upgraded to the 0.5.0 version of this plugin and am seeing an error when running `vagrant help`

```
> vagrant help
/Users/cbaldauf/.vagrant.d/gems/gems/vagrant-managed-servers-0.5.0/lib/vagrant-managed-servers/plugin.rb:27:in `require_relative': cannot load such file -- /Users/cbaldauf/.vagrant.d/gems/gems/vagrant-managed-servers-0.5.0/lib/vagrant-managed-servers/command/reboot (LoadError)
	from /Users/cbaldauf/.vagrant.d/gems/gems/vagrant-managed-servers-0.5.0/lib/vagrant-managed-servers/plugin.rb:27:in `block in <class:Plugin>'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/cli.rb:75:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/cli.rb:75:in `block (2 levels) in help'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/registry.rb:49:in `block in each'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/registry.rb:48:in `each'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/registry.rb:48:in `each'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/cli.rb:69:in `block in help'
	from /opt/vagrant/embedded/lib/ruby/2.0.0/optparse.rb:885:in `initialize'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/cli.rb:57:in `new'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/cli.rb:57:in `help'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/cli.rb:32:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:301:in `cli'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/plugins/commands/help/command.rb:11:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/cli.rb:42:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.7.2/lib/vagrant/environment.rb:301:in `cli'
	from /opt/vagrant/bin/../embedded/gems/gems/vagrant-1.7.2/bin/vagrant:174:in `<main>'
```

Looking at the 0.5.0 tag, it does look like that file isn't present, but is being required in plugin.rb.

https://github.com/tknerr/vagrant-managed-servers/tree/v0.5.0/lib/vagrant-managed-servers
https://github.com/tknerr/vagrant-managed-servers/blob/v0.5.0/lib/vagrant-managed-servers/plugin.rb#L27

@jdaviscooke

**Comment by tknerr** [comment]
@chrisbaldauf thanks for finding this one. I was actually a bit surprised because I had never seen that while I was testing #31. Also the "acceptance tests" (`rake acceptance`) were passing, so this one really slipped through... ;-)

I can easily reproduce this, but only for `vagrant help` (edited title accordingly). As you pointed out it's obvious why. Let me fix it and I'll push a new release immediately...

**Comment by tknerr** [comment]
Bugfix released as version 0.5.1. Thanks for reporting!

**Comment by chrisbaldauf** [comment]
Confirmed that this is fixed in 0.5.1 and that it installs that version by default. Thanks for the quick turnaround!
